src/roadmap-goals.md

```diff
@@ -0,0 +1,119 @@
+# Roadmap & Goals
+
+the roadmap gives you an overview and orientation for your first weeks at VI.
+
+## Goal
+> Every new frontend developer is able to implement components in our Biotope projects, in compliance with our VI standards!
+
+## Roadmap
+
+The following part is an overview of all the topics and challanges, that you are going to pass the next view weeks.
+
+
+<roadmap-component>
+
+### Preparation
+
+You will be assigned a mentor and also buddies for each lesson.
+You have to organize your accounts and tools and afterwards this roadmap will be adjust to your individual skills. Beside that you will also get an introduction to the development manifesto.
+
+*Goal:you can start right away and know where you are heading without being over- or underchallenged*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Getting started
+
+The first part of your journey is to get an overview of the topics, that come across during your first weeks. You will learn which vision the onboarding process follows and get in touch with our toolset like JIRA and GIT/BITBUCKET
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental I
+In that first part of the fundamentals of frontend development, you will be shown how to handle the mobile first approach, html semantics and css. Beside that you also get knowledge about accessability with WAI-ARIA and at least how to handle the SVG Icons.
+
+__Challanges__
+- Page Layout
+- Content Structure and Webfonts
+- Navigation
+- Responsive Webdesign
+- SVG Icons
+
+*Goal: you know how to create a modern static web page*
+
+</roadmap-component>
+
+<roadmap-component>
+
+### Fundamental II
```

Comment:
We have FundamentalS everywhere, so we should do it here too I guess

Reply:
fixed